.settings/org.eclipse.wst.common.component

```diff
@@ -6,8 +6,6 @@
                 
         <wb-resource deploy-path="/" source-path="/src/main/webapp" tag="defaultRootSource"/>
                 
-        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/java"/>
```

Comment:
did you delete a source folder?

Reply:
I deleted a JSP file that was deprecated and updated pathways. Web-inf/classes doesn't exist anymore.